modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/suggestions/display/context/SuggestionsPortletDisplayContextBuilderTest.java

```diff
@@ -57,7 +58,8 @@ public void setUp() {
 		MockitoAnnotations.initMocks(this);
 
 		_setUpHtml();
-		_setUpHttp();
+
```

Comment:
@BryanEngler I changed the logic for this test to avoid mocking static method, the old logic does not make sense to me as we won't get a result of search url like that in production, could you please check if this change is OK?

Thanks.
Tina.